dartagnan/src/main/java/com/dat3m/dartagnan/Dartagnan.java

```diff
@@ -398,6 +409,45 @@ public static String generateResultSummary(VerificationTask task, ProverEnvironm
         return summary.toString();
     }
 
+    private static void increaseBoundAndDump(Event ev, Configuration config) {
+
+        String evId = String.valueOf(ev.getMetadata(UnrollingId.class).value());
+        String incBound = String.valueOf(ev.getMetadata(UnrollingBound.class).value() + 1);
+
+        // We read from and write to the same CSV file,
+        // thus we need to split this in two loops
+        List<String[]> modifiedRecords = new ArrayList<>();
+        // We read the file written by the LoopUnrolling pass,
+        // thus we use BOUNDS_SAVE_PATH also for the reader
+        try (Reader reader = new FileReader(config.hasProperty(BOUNDS_SAVE_PATH) ?
+                                            config.getProperty(BOUNDS_SAVE_PATH) :
+                                            GlobalSettings.getBoundsFile())) {
+            for (CSVRecord record : CSVFormat.DEFAULT.parse(reader)) {
```

Comment:
I don't like using the deprecated methods here, but I could not find a cleaner alternative. Any ideas?